### ISSUE TITLE
About page is empty

### ISSUE THREAD

**Issue by Mathis-Z** [openHPI/codeharbor#1219]
There is currently no text on the about page. Was there something in the past? :thinking:

**Comment by MrSerth** [comment]
> Was there something in the past? 🤔

No, there wasn't yet. When the about page was [added](https://github.com/openHPI/codeharbor/commit/2984aece815176f94523afba9b9d64300c1baf7b#diff-44438ce218f5287c58d0017f965d888715635d94280669896f75841fbd7b4cd7R403-R405), the paragraph was empty. Hence, that's nothing we broke but some missing "feature". I agree, that we should either remove the about page or add some real content.

**Comment by Mathis-Z** [comment]
I think an about page is not really required because we already have the home page which fills this role. Another about page would probably just repeat much of that information.
